docs/README.md

```diff
@@ -228,6 +229,21 @@ The valid values for this option are:
 * If you configure the LogDNA Agent to run as non-root, review the [documentation](KUBERNETES.md#enabling-file-offset-tracking-across-restarts) about enabling "statefulness" for the LogDNA Agent.
 * When upgrading from LogDNA Agent version 3.0 to 3.1, the state file will initially be empty, so the lookback setting will be used for existing files. After that (i.e. on process restart), the state file will be present and will be used.
 
+### Configuring Lease Startup
+
+The lease startup configuration uses Kubernetes Leases to limit the number of agents that can start at one time on a cluster. When enabled, the agent will "claim" a lease before starting. Once started, the agent will then release the lease. If no leases are available, the agent will wait for one to become available. This feature would only be needed if running the agent on a cluster large enough that you'd risk crashing `etcd` if all the the agents tried to connect at once.
+
+The valid values for this option are:
+* When set to **`None`** (default):
+  * The agent will start normally and will not attempt to obtain a lease before starting.
+* When set to **`limited`**:
```

Comment:
@c-nixon I changed this from "on" to "limited" because when @jiangao1 and I were running some test it looked like "on" has some special meaning in yaml on it need to be double quoted, otherwise it threw an error when you tried to apply it. I changed to remove any confusion but let me know if you think another option name would be better?

Reply:
Hmm, yeh that's annoying about "on". Although I think "limited" might carry the wrong emphasis... I think we really we mean "opportunistic" but that's also quite long... maybe "IfAvailable"? Or maybe "attempt" and "require" instead of "always"?

@alfeng6 What do you think?

Reply:
I like "attempt" and "require" but let's see what @alfeng6 says.

Reply:
I like "attempt" and "require" as well — the word gives the notion that we will actually try something.

Reply:
"attempt" makes me think it will try a few times. "Always" makes me think it'll always keep trying.

Reply:
Updated.